Formula/xonsh.rb

```diff
@@ -35,14 +35,9 @@ class Xonsh < Formula
     sha256 "6283b7a58477dd8478fbb9e76defb37968ee4ba47b05ec1c053cb39638bd7398"
   end
 
-  resource "six" do
-    url "https://files.pythonhosted.org/packages/21/9f/b251f7f8a76dec1d6651be194dfba8fb8d7781d10ab3987190de8391d08e/six-1.14.0.tar.gz"
-    sha256 "236bdbdce46e6e6a3d61a337c0f8b763ca1e8717c03b369e87a7ec7ce1319c0a"
-  end
```

Comment:
This was deliberately removed. I checked with the Xonsh maintainers directly and it's no longer a dependency: https://github.com/xonsh/xonsh/issues/3722#issuecomment-681457253